bpf/rtt_tracker.h

```diff
@@ -62,6 +97,9 @@ static __always_inline void calculate_flow_rtt_tcp(pkt_info *pkt, u8 direction,
                 if (trace_messages && ret != 0) {
                     bpf_printk("error evicting flow sequence: %d", ret);
                 }
+                // This is an ACK packet with valid sequence id so a SYN must
```

Comment:
how are u sure this is an `ack` w/o inspecting the TCP flags ?

Reply:
This is contained in the ` if (IS_ACK_PACKET(pkt)) {` check